scripts/deploy-to-gh-pages.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Pull requests and commits to other branches shouldn't try to deploy
+if [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ "$TRAVIS_BRANCH" != "jsdoc-ghpages" ]; then
```

Comment:
"jsdoc-ghpages"? It should say "v4", right?

Reply:
Yes, before merging